src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -82,7 +83,7 @@ private ResponseEntity postEventInternal(final String eventTypeName,
             final JSONArray eventsAsJsonObjects = new JSONArray(eventsAsString);
 
             final int eventCount = eventsAsJsonObjects.length();
-            eventTypeMetrics.reportSizing(eventCount, eventsAsString.length());
+            eventTypeMetrics.reportSizing(eventCount, eventsAsString.getBytes(Charsets.UTF_8).length);
```

Comment:
I think it's a bit more complicated than that: if we return a 207 then we need to re-compute the sum of the sizes of the events published successfully, as well as their number. However, on second thought, I agree that it makes more sense to fix it here.